src/group_theory/nilpotent.lean

```diff
@@ -891,4 +892,21 @@ begin
   exact nilpotent_of_mul_equiv e,
 end
 
+/-- A finite group is nilpotent iff the normalizer condition holds, and iff all sylow groups are
+normal and iff its the direct product of its sylow groups.  -/
```

Comment:
I think this docstring is missing the third item in the `tfae`. Intentional?

Reply:
Good catch; I added that a bit later to the theorem and (of course) failed to update the docstring. fixed.